tests/components/input_select/test_init.py

```diff
@@ -400,5 +440,159 @@ def select_previous(hass, entity_id):
     assert state_1 is None
     assert state_2 is not None
     assert state_3 is not None
-    assert "reloaded option" == state_2.state
+    assert "an option" == state_2.state
     assert "newer option" == state_3.state
+    assert ent_reg.async_get_entity_id(DOMAIN, DOMAIN, "test_1") is None
+    assert ent_reg.async_get_entity_id(DOMAIN, DOMAIN, "test_2") is not None
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4803353)